Dockerfile

```diff
@@ -7,3 +7,4 @@ RUN /bin/herokuish buildpack install \
 	&& ln -s /bin/herokuish /exec
 COPY include/default_user.bash /tmp/default_user.bash
 RUN bash /tmp/default_user.bash && rm -f /tmp/default_user.bash
+RUN apt-get update && apt-get install -y daemontools ruby-dev
```

Comment:
Can probably move this up and run it as the second step - but also clear out apt cache?

Reply:
@pynixwang todos:
- move `RUN apt-get...` above `RUN curl..`
- change the line to `RUN apt-get update -qq && apt-get install -yy -q daemontools ruby-dev`
- add `RUN apt-get clean && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*` to the bottom